code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -267,19 +301,27 @@
 	result = /obj/item/food/snacks/cubancarp
 
 /datum/recipe/microwave/popcorn
-	items = list(/obj/item/food/snacks/grown/corn)
+	items = list(
+		/obj/item/food/snacks/grown/corn
+	)
 	result = /obj/item/food/snacks/popcorn
 
 /datum/recipe/microwave/spacylibertyduff
 	reagents = list("water" = 5, "vodka" = 5)
-	items = list(/obj/item/food/snacks/grown/mushroom/libertycap, /obj/item/food/snacks/grown/mushroom/libertycap,
-				/obj/item/food/snacks/grown/mushroom/libertycap)
```

Comment:
i think this was the single worst list in the code

Reply:
It was one of the lists of all time.